### DISCUSSION TITLE
Still working?

Hi! 

This looks super useful and I want to use it, but I can't run example cases currently. I have tried git and pip installs. I am using Ubuntu 24.04, Python 3.11, miniconda3, and the latest openmc. Could be just me, but thought I'd ask. Thanks! 

Oliver

### DISCUSSION THREAD

**Discussion by o1ivr** [arfc/rollo#61] [Q&A]
![received_1552357348969346.png](https://github.com/user-attachments/assets/a1cf094b-c86f-48cb-b8bd-a916d1c1a1f9)

![received_1155200068872389.png](https://github.com/user-attachments/assets/b02949e6-d2c7-4dc6-a818-12507df63c83)

![received_1007313801072946.png](https://github.com/user-attachments/assets/9f3b1ceb-c7cf-4ab2-abb6-e26f9658826c)

![received_409426408788786.png](https://github.com/user-attachments/assets/9ec44ead-de22-4e0d-8a39-6a2bf293f452)

**Comment by katyhuff** [comment]
Rather than screenshots, it is helpful if you copy the text into a code snippet. 

However, your snippets seem to indicate that your issue may be associated with failure to install mpi4py. Another possible issue is that it's generally best to run a tool like this from outside the source code directory. 

Please try both and let me know how it goes.
